cluster/examples/kubernetes/ceph/cluster-on-local-pvc.yaml

```diff
@@ -231,6 +231,8 @@ spec:
               volumeMode: Block
               accessModes:
                 - ReadWriteOnce
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement
+    onlyApplySpecificOSDPlacement: false
```

Comment:
I believe the default behavior that we are trying to imply is to keep `onlyApplySpecificOSDPlacement: false`

So should we comment this field out by default in all the example CRs?  Users can remove the comment and mark it as true if don't want to merge. 

@travisn  @subhamkrai

Reply:
Right, the default is false after upgrades, and also I think it's good if the default false shows uncommented. Then it's easy to change to true. But I know we've used the pattern you're suggesting in other places too.